packages/web-app/src/hooks/useCachedDao.tsx

```diff
@@ -0,0 +1,102 @@
+import {isAddress} from 'ethers/lib/utils';
+import {useCache} from './useCache';
+import {useClient} from './useClient';
+import {
+  ICreateDaoERC20Voting,
+  ICreateDaoWhitelistVoting,
+} from '@aragon/sdk-client';
+import {Dao} from 'utils/types';
+
+interface useDaoResponse {
```

Comment:
By convention, TS interfaces begin with a capital `i` and are camelcase
```suggestion
interface IUseDaoResponse {
```